README.md

```diff
@@ -2,6 +2,96 @@
 
 This is the web UI for the SCS statuspage.
 
-Right now, it only contains the HTML prototype used to develop the design, layout and necessary features. It will soon be joined by an actual Angular implementation.
+**This project is currently in the technical preview stage. Do not use in production.**
+
+The status page is a simple web application that allows users to check if all services and components of a SCS installation are currently in a working state. Furthermore, it allows to read updates on ongoing and past incidents.
+
+## Using the Status Page
+
+To use the status page, you have to customize the configuration file and then create a container to use in your SCS setup.
+
+### Prerequisites
+
+Your SCS setup needs to include an instance of the [Status Page API server](https://github.com/SovereignCloudStack/status-page-api).
+
+### Customizing the Configuration
+
+You will find a configuration file named `config.json` in the `src/assets` directory. This file looks like this (sans comments, for the JSON gods abhor them):
+
+```json
```

Comment:
```suggestion
```json5
```

Allows comments

Reply:
Does the JSON parser used by Angular allow JSON5? If not, we should take care not to insinuate that JSON5 would be a viable option. Need to check that.

Reply:
just for the code highlight in github... its ja readme.

default json does not allow comments.

Reply:
Okay. Even though I found a Github issue that referenced Angular accepting JSON5, it does not do so in my dev setup.